pkg/repositories/transformers/constants.go

```diff
@@ -0,0 +1,3 @@
+package transformers
+
+const Outputs = "offloaded_outputs"
```

Comment:
What does this `Outputs` const represent? maybe a slightly more descriptive name helps? or a comment line?
something like this?
```suggestion
const OutputsObjectSuffix = "offloaded_outputs"
```

Reply:
done